{{cookiecutter.repo_name}}/development.ini

```diff
@@ -24,6 +24,10 @@ sqlalchemy.url = sqlite:///%(here)s/{{ cookiecutter.repo_name }}.sqlite
 # wsgi server configuration
 ###
 
+[alembic]
+# path to migration scripts
+script_location = {{ cookiecutter.repo_name }}/alembic
+
```

Comment:
It is OK with me, is there maybe any other default value that can be changed? I wanted to put bare minimum in template rest left to user, maybe add this as commented?